models/ldndc/inst/setup_template.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
when you move a file or change its name, you should do a `git mv` rather than deleting the old file and adding a new one. This preserves the version control (which otherwise gets broken) and reduces repository bloat